arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/ContextFragment.java

```diff
@@ -93,7 +93,7 @@ public static ContextFragment executionProvidesEmptyReturnData(final Hub hub) {
 
   public static ContextFragment executionProvidesEmptyReturnData(final Hub hub, int contextNumber) {
     CallStack callStack = hub.callStack();
-    int parentId = callStack.getByContextNumber(contextNumber).callerId();
+    int parentId = callStack.getByContextNumber(contextNumber).parentId();
```

Comment:
`callerId -> parentId` since this `caller` isn't the right term in case of a deployment and also we constantly use the term "child" so `parent` is appropriate.